api/db/vars.env

```diff
@@ -9,3 +11,6 @@ LIQUIBASE_DB_PASSWORD=lb-notasecret
 MYSQL_ROOT_PASSWORD=root-notasecret
 WORKBENCH_DB_USER=workbench
 WORKBENCH_DB_PASSWORD=wb-notasecret
+# TODO: change to public DB user once it exists
+PUBLIC_DB_USER=workbench
+PUBLIC_DB_PASSWORD=workbench
```

Comment:
Since  you can specify the env file in the docker-compose.yml , I might make public have it's own public_vars.env file. This could save a few code changes and just use same var names everywhere.  May look better for security too if the public api never reads any private vars and passwords and connection info.